docset/winserver2022-ps/failoverclusters/Remove-ClusterResourceDependency.md

```diff
@@ -34,27 +34,26 @@ offline might vary.
 
 ### Example 1
 
-```
-PS C:\> Remove-ClusterResourceDependency -Resource cluster1FS -Provider "IP Address 2001:4898:9:2:: (3)"
-Name                State               Group               ResourceType 
-----                -----               -----               ------------ 
-cluster1FS          Online              cluster1FS          Network Name
+```powershell
+Remove-ClusterResourceDependency -Resource cluster1FS -Provider "IP Address 2001:4898:9:2:: (3)"
```

Comment:
Should this example use splatting like the one after it?
```suggestion
```powershell
$parameters = @{
    Resource = 'cluster1FS'
    Provider = 'IP Address 2001:4898:9:2:: (3)'
}
Remove-ClusterResourceDependency @parameters
```